backend/clickhouse/log_row.go

```diff
@@ -105,6 +105,9 @@ func WithSeverityText(severityText string) LogRowOption {
 func WithSource(source modelInputs.LogSource) LogRowOption {
 	return func(l *LogRow) {
 		l.Source = source
+		if l.Source == "" {
```

Comment:
What do you think about making this:

```go
func WithSource(source modelInputs.LogSource) LogRowOption {
	return func(l *LogRow) {
		if l.Source == modelInputs.LogSourceFrontend {
      l.Source = modelInputs.LogSourceFrontend
		} else {
      l.Source = modelInputs.LogSourceBackend
		} 
	}
}
```

Per this [comment](https://github.com/highlight/highlight/pull/4839/files#r1158714695), we don't have a true union type but this type of logic would us basically there.

My thoughts are that if anyone passes "source = foo", we should just ignore it and default to "backend"

Reply:
sure!